composer.json

```diff
@@ -18,6 +18,8 @@
     "require-dev": {
         "zendframework/zend-diactoros": "^1.0",
         "guzzlehttp/psr7": "^1.0",
+        "ext-zlib": "*",
+        "clue/stream-filter": "^1.3",
```

Comment:
lets add these to suggest, and explain for what case one would want to use them in a project.